# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogGps.h
# SbgLogGpsRaw
uint8[4086] rawBuffer
uint32 bufferSize